packages/manager/apps/dedicated/client/app/dedicated/server/index.js

```diff
@@ -0,0 +1,66 @@
+import angular from 'angular';
+
+import '@uirouter/angularjs';
+import 'angular-translate';
+import 'ovh-ui-angular';
+
+import bandwidth from './bandwidth/bandwidth.module';
+import dashboard from './dashboard';
+import interfaces from './interfaces/interfaces.module';
+import monitoring from './monitoring';
+import servers from './servers/servers.module';
+
+import bandwidthVrackOrderService from './server.bandwidth-vrack-order.service';
+import component from './server.component';
```

Comment:
This is personal preference but I like having imports like component and routing split from other module declaration (that way I easily can find what my module is declaring and what other modules are required). There is no official convention about this, no tooling (because ideally it should be done automatically), it is just an idea :)